apps/dashboard/src/components/mission/DesireSection.tsx

```diff
@@ -19,7 +13,7 @@ const DesireSection = () => {
       >
         thirdweb will...
       </Heading>
-      <Flex maxW="2xl">
```

Comment:
Are we sure that chakra's 2xl is same as tailwind 2xl. 

It doesn't look like that's the case 
chakra - 96em  (1536px)
https://v2.chakra-ui.com/docs/styled-system/responsive-styles

tailwind - 42rem; (672px)
https://tailwindcss.com/docs/max-width

Reply:
oops, going over the other changes. ty

Reply:
This belongs to a marketing page so probably just gonna hard-code it to 96rem.

Reply:
sounds good, I think we don't even need to make changes in marketing pages - we'll move them to framer